### ISSUE TITLE
Change color of unavailable song

### ISSUE THREAD

**Issue by ThomasFrans** [hrkfdn/ncspot#1315]
**Is your feature request related to a problem? Please describe.**
The main branch just got a feature that shows when a song isn't available. Currently the song is displayed in the same color as the currently playing one, which is a bit confusing.

**Describe the solution you'd like**
Use a color that isn't used yet. Maybe also use a more 'neutral'/less saturated color, since that suggests it can't be played (like hidden files in a file manager or grayed out buttons in a GUI program). Maybe make it configurable so users can change it with the theme, but the default is fine for users who don't like to change it explicitly.

**Describe alternatives you've considered**
\/

**Additional context**
Just opened this for discussion. I'd do the change but I wanted to ask first.

**Comment by hrkfdn** [comment]
Hmm, could that be a theming thing? It's not the same color for me.

![image](https://github.com/hrkfdn/ncspot/assets/19884/e44112d6-60f8-4055-a873-e6b7f2778efb)

**Comment by ThomasFrans** [comment]
Oh weird, I ran both with and without the config (`--config=/dev/null`) and it was always the same color as the currently playing track. It looks like this for me:
![image](https://github.com/hrkfdn/ncspot/assets/48214567/2e8c06fd-6c44-4ad1-9d49-6484f46fd9c4)

One of them is the currently playing track, and the other is an unavailable one. Maybe it's because I've defined a color scheme for the terminal or something? I would indeed have expected it to look like your terminal where it is less saturated.
